### ISSUE TITLE
Is the acc of session 0 for CUB200 obtained at 500 epochs?

### ISSUE THREAD

**Issue by WanyiLi** [TouqeerAhmad/FeSSSS#2]
We reproduced the experiment and found:
(1) For CUB200 data set, the acc of session 0 is 80.17%,  when using the settings in the paper, i.e. the classification module is trained for 1000 epochs at a learning rate of 0.1 for the base session.  It is better than the paper reported 79.60%.  
So we guess that, the reported accuracy 79.60% of session 0 is obtained from classification module trained for 500 epochs.

(2) We check the acc at 500 epochs, as expected, it is 79.61%, very close to the reported.

Is the acc of session 0 for CUB200 obtained at 500 epochs?
